packages/e2e-test-utils-playwright/src/types.ts

```diff
@@ -3,6 +3,23 @@ declare global {
 		// Silence the warning for `window.wp` in Playwright's evaluate functions.
 		wp: any;
 	}
+
+	interface LayoutShiftAttribution {
+		readonly node: Node;
+		readonly previousRect: DOMRectReadOnly;
+		readonly currentRect: DOMRectReadOnly;
+		readonly toJSON: () => string;
+	}
+	interface LayoutShift extends PerformanceEntry {
```

Comment:
Nit: Let's add some comments above pointing them to the MDN doc or something similar and note that these are still experimental web interfaces that might change in the future.